rele/client.py

```diff
@@ -42,9 +42,15 @@ def __init__(self, gc_project_id, credentials, default_ack_deadline=None):
         self._ack_deadline = default_ack_deadline or DEFAULT_ACK_DEADLINE
 
         if USE_EMULATOR:
-            self._client = pubsub_v1.SubscriberClient()
+            self._subscriber_client = pubsub_v1.SubscriberClient()
+            self._publisher_subscriber_client = pubsub_v1.PublisherClient()
         else:
-            self._client = pubsub_v1.SubscriberClient(credentials=credentials)
+            self._subscriber_client = pubsub_v1.SubscriberClient(
+                credentials=credentials
+            )
+            self._publisher_subscriber_client = pubsub_v1.PublisherClient(
+                credentials=credentials
+            )
```

Comment:
The name `_publisher_subscriber_client` seems confusing to me. It looks like it's mixing opposed terms: publishers and subscribers.
Why did you choose that name? Have you thought about renaming it to `_publisher_client`? 🤔

Reply:
It was a mistake. Fixed.